src/Storefront/Resources/views/storefront/utilities/thumbnail.html.twig

```diff
@@ -33,6 +33,10 @@
         {% set attributes = attributes|merge({ loading: 'eager' }) %}
     {% endif %}
 
+    {% if attributes.fetchpriority == 'high' %}
+        {% set attributes = attributes|merge({ 'fetchpriority': 'high' }) %}
+    {% endif %}
```

Comment:
Do we actually need this condition?

Reply:
Yes definitely! Without this condition every image would always get `fetchpriority="high"`, which is not something you want. Always having `fetchpriority="high"` on an image can potentially be bad for performance.

Reply:
I also meant the content inside. 

You check for the key fetchpriority = high and assign fetchpriority = high
You don't need to assign something what already there 😄 

Shouldn't it also work without changes on the thumbnail file?

Reply:
Hmm good point! 😅 
I'll change it!